integration-tests/bats/primary-key-changes.bats

```diff
@@ -0,0 +1,539 @@
+#!/usr/bin/env bats
```

Comment:
need to add a test here for failing to add a PK because the column has duplicates